docs/lifecycle-events.md

```diff
@@ -25,7 +25,9 @@ Fired before the element is removed from the DOM.
 Note that when using this event you are responsible for removing the element yourself.
 
 ```js
-element.parent.removeChild(element)
+if (element.parentNode) {
```

Comment:
We can certainly remove that guard statement, I just copied it from MDN.

I was wondering if it could still be useful, though, depending on what you're doing in the `onremove` hook.? Is there a chance that a node would be orphaned if you're doing something async in `onremove`, and patch removes it parent?  Or will the parent node stick around until the child gets removed?

For example, what would happen to the `div` if i delay the `h1` removal on a route change?
```
view: {
  '/': () => h('div', {}, [
    h('h1', {onremove: somethingAsyncThatTakesAwhile}, 'Test')
  ]),
  '/about': () => h('p', {}, 'About page')
}
```